packages/nimble-components/src/wafer-map/tests/wafer-map.stories.ts

```diff
@@ -90,7 +90,7 @@ const metadata: Meta<WaferMapArgs> = {
             }
         },
         actions: {
-            handles: ['click', 'die-selected']
+            handles: ['click', 'die-hover']
```

Comment:
A new pattern we've been trying out is to add items to the Controls section for each of the events that the component fires. This give us a place to explicitly mention the events and describe their event data. You can see an example of this in how the story for the Banner component documents the `toggle` event.

Unless you have an objection, want to try the same thing for this new event? The docs could say something like "The `die-hover` event is fired whenever the mouse enters or leaves a die. In the event data, `detail.currentDie` will be set to the element of the `dies` array that is being hovered or `undefined` if the mouse is leaving a die."

<img width="782" alt="image" src="https://user-images.githubusercontent.com/10500124/221619430-a87c6867-06af-4c83-a247-65aeeef7ec0b.png">